pkg/etcd/init/init.go

```diff
@@ -0,0 +1,121 @@
+package init
+
+import (
+	"context"
+	"fmt"
+
+	"go.etcd.io/etcd/client/v3"
+)
+
+const (
+	rootRoleName = "root"
+)
+
+func InitEtcd(ctx context.Context, client *clientv3.Client, clusterName string) error {
+	_, err := client.Put(ctx, "cilium/.has-cluster-config", "true")
```

Comment:
I think it could make sense to add a constant for this key value. Also, I'm assuming that the goal is to eventually use this to stop init if this key is found?